site/content/en/blog/releases/1.27.0-rc.md

```diff
@@ -0,0 +1,81 @@
+
+**Fixed bugs:**
+
+- Flaky test: CPP SDK Conformance tests [\#2298](https://github.com/googleforgames/agones/issues/2298)
+
+**Closed issues:**
+
+- Replace uses of the io/ioutil package [\#2748](https://github.com/googleforgames/agones/issues/2748)
+- Release 1.26.0 [\#2737](https://github.com/googleforgames/agones/issues/2737)
+- Log cleanup: Verbose error log on pod not yet running [\#2665](https://github.com/googleforgames/agones/issues/2665)
+- Upgrade gRPC version [\#1797](https://github.com/googleforgames/agones/issues/1797)
+
+**Merged pull requests:**
+
+- Add repository for hashicorp/consul, and disable the consul-consul-client [\#2764](https://github.com/googleforgames/agones/pull/2764) ([gongmax](https://github.com/gongmax))
+- 2665 Log cleanup: Verbose error log on pod not yet running [\#2763](https://github.com/googleforgames/agones/pull/2763) ([mangalpalli](https://github.com/mangalpalli))
+- Enable image streaming for e2e test cluster [\#2762](https://github.com/googleforgames/agones/pull/2762) ([gongmax](https://github.com/gongmax))
+- Fix case of HTTP module reference [\#2760](https://github.com/googleforgames/agones/pull/2760) ([TroutZhang](https://github.com/TroutZhang))
+- Add an example of setting autoscaler behavior in a Fleet. [\#2759](https://github.com/googleforgames/agones/pull/2759) ([roberthbailey](https://github.com/roberthbailey))
+- Enable image streaming everywhere by default [\#2756](https://github.com/googleforgames/agones/pull/2756) ([gongmax](https://github.com/gongmax))
+- Update the Kubernetes upgrade instructions to include instructions for upgrading gRPC [\#2755](https://github.com/googleforgames/agones/pull/2755) ([roberthbailey](https://github.com/roberthbailey))
+- If the user has specified cluster autoscaling behavior for their gameserver then don't overwrite it [\#2754](https://github.com/googleforgames/agones/pull/2754) ([roberthbailey](https://github.com/roberthbailey))
+- Replace uses of the io/ioutil package [\#2752](https://github.com/googleforgames/agones/pull/2752) ([gongmax](https://github.com/gongmax))
+- Bump the example images version [\#2751](https://github.com/googleforgames/agones/pull/2751) ([gongmax](https://github.com/gongmax))
+- Upgrade Go version to 1.19.1 [\#2743](https://github.com/googleforgames/agones/pull/2743) ([gongmax](https://github.com/gongmax))
+- Added a title to the 1.26.0 release [\#2742](https://github.com/googleforgames/agones/pull/2742) ([markmandel](https://github.com/markmandel))
+- updates for upcoming release [\#2741](https://github.com/googleforgames/agones/pull/2741) ([SaitejaTamma](https://github.com/SaitejaTamma))
+- Allocator Informer Event  optimize updateFunc [\#2731](https://github.com/googleforgames/agones/pull/2731) ([alvin-7](https://github.com/alvin-7))
+
+
+Documentation: https://development.agones.dev/site/
+
+
+See <a href="https://github.com/googleforgames/agones/blob/release-1.27.0-rc/CHANGELOG.md">CHANGELOG</a> for more details on changes.
```

Comment:
Without data-proofer-ignore this is going to fail. See the template: https://raw.githubusercontent.com/googleforgames/agones/main/docs/governance/templates/release.md